build.sh

```diff
@@ -231,7 +231,7 @@ build_native()
             extraCmakeArguments="$extraCmakeArguments -DCLR_CMAKE_ENABLE_CODE_COVERAGE=1"
         fi
 
-        nextCommand="\"$scriptDir/gen-buildsys.sh\" \"$__ProjectRoot\" \"$intermediatesForBuild\" $platformArch $__BuildType $generator $scan_build $extraCmakeArguments $__cmakeargs"
+        nextCommand="CONFIG_DIR=\"$__RepoRootDir/eng/common/cross\" \"$scriptDir/gen-buildsys.sh\" \"$__ProjectRoot\" \"$intermediatesForBuild\" $platformArch $__BuildType $generator $scan_build $extraCmakeArguments $__cmakeargs"
```

Comment:
I don't see the CONFIG_DIR being used in the gen-buildsys.sh, so it is not clear to me why we are setting it for the script. The same in the build-script.sh

Reply:
Ahh, right you are, my bad, I already cleaned this up in CoreCLR, this change doesn't need porting from the runtime repo. Will remove, thanks for spotting!

Reply:
Deleted in 2nd commit.